notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<ul><li>On your second bullet point ("Narrow down a list..."), I'd suggest rewording slightly: "Find the binary candidates most likely to be observed as eclipsing binaries by using a set of parameters from the APOGEE catalog."</li><li>On your third bullet point ("Determine if a stellar system..."), slight rewording: "Determine if a candidate system in the APOGEE catalog has a light curve in (a) the TESS Data for Asteroseismology Lightcurves archive from TASOC (the TESS Asteroseismic Science Operations Center) or (b) the TESS Lightcurves From The MIT Quick-Look Pipeline ("QLP") archive using&nbsp;<code>astroquery</code>&nbsp;and the MAST archive." -- Note, spelling correction of "Asteroseismic".</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>